dynamic_values/config_modules/service_authorizations/service_authorizations.tf

```diff
@@ -59,7 +59,7 @@ module "kms_to_block_storage" {
   ]
 }
 
-# workaround for https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/issues/645
+# Required service authorization access policy for Kubernetes Service and the KMS provider
```

Comment:
Akash I thought we said we would add some extra info in the comment as to why it is required. That basically it only gets auto created if doing cluster data encryption, however it needs to exist before the if doing boot volume encryption, so hence we need to explicitly create it

Reply:
@ocofaigh I have updated the comment, please check.